tests/dataframe_test.py

```diff
@@ -387,6 +387,11 @@ def test_gb_series(self):
         self.assertListEqual(c.index.to_list(), ["Bob", "Alice", "Carol"])
         self.assertListEqual(c.values.to_list(), [2, 3, 1])
 
+        # testing counts with IPv4 column
+        s = ak.DataFrame({'a': ak.IPv4(ak.array([1, 2, 3, 4]))}).groupby('a').count()
+        pds = pd.Series(data=np.array([1, 1, 1, 1]), index=pd.Index(data=np.array(['0.0.0.1', '0.0.0.2', '0.0.0.3', '0.0.0.4'], dtype="<U7")))
```

Comment:
Small style suggestion, change to ak.ones(4)

Reply:
sorry, *np.ones(4)

Reply:
@jaketrookman - I updated this to `np.ones(4, dtype=np.int64)`. `np.ones` defaults the `dtype` to `float64`.